static/js/biomedical/disease/page.tsx

```diff
@@ -101,85 +113,109 @@ export class Page extends React.Component<PagePropType, PageStateType> {
       log: false,
       name: "diseaseData",
     };
-    // Render the graphs only when the data is not null
-    if (this.state.data === null) {
```

Comment:
nit: I think we should check that the data has been fetched before rendering the page. You can do this by adding one more field in the state 'dataFetched' & checking that field before rendering the page.

Reply:
Added the dataFetched field and set it to true when the rest of the fields are fetched!